center/src/main/resources/application.yml

```diff
@@ -55,7 +55,7 @@ app:
     type: ${STORAGE_TYPE:LOCAL} # current available options: LOCAL (default), AZURE
     azure:
       connection: ${BLOB_CONNECTION_STR:}
-      fileLimitDay: 6
+      fileLimitDay: ${FILE_LIMIT_DAY:6}
```

Comment:
Suggest change a better name for this property: fileExpiryDay cc @zhou9584

Reply:
Resolved for now. Will update in another PR specifically.